api/v1/paths/members/{memberId}/games.js

```diff
@@ -3,7 +3,6 @@ const appRoot = require('app-root-path');
 const gameDao = require('../../../db/oracledb/game-dao');
 
 const { errorBuilder, errorHandler } = appRoot.require('errors/errors');
-const { openapi: { paths } } = appRoot.require('utils/load-openapi');
```

Comment:
Why'd you remove this line? The api crashes without it.

Reply:
I tried to pull the changes from skeleton to the API, and this line is not required. I merged another branch to this branch so the problem can be fixed.